src/app/skill.service.ts

```diff
@@ -8,6 +8,9 @@ import { BehaviorSubject } from 'rxjs';
 })
 
 export class SkillService {
+  all() {
+    throw new Error("Method not implemented.");
```

Comment:
I suppose this is fine for now, but this will of course need to be implemented once the backend is updated.